docs/README.md

```diff
@@ -0,0 +1,48 @@
+# CDEvents Implementation
+
+The implementation of CDEvents is use case driven. We focus on a few use cases that demonstrate the business value of interoperability in the CI/CD space and for each of them we identify a target architecture, its logical building blocks and a set of software components that can be used to implement it.
+
+Some of the software required is part of the CDEvents project and other is external. All proposed implementation focus exclusively on open source software components. Commercial software that implements standard protocols like CDEvents can be slotted in to replace or complement the open source software adopted by the CDEvents Implementation WG.
+
+## Use Cases
+
+CDEvents use cases may apply to two personas: open source project communities and organizations doing in-house software development.
+The requirements and design of the implementation vary based on the persona and are documented in the use-case specific document.
+
+- [Enhance Security Posture through Automation](./security_automation.md)
+- Improve Developer Experience through Consistency and Visibility (TBD)
+- Drive Efficiency through DevOps Metrics (TBD)
+
+## Common Components
+
+### Messaging Platform
```

Comment:
I think a links service is going to be needed as well, and I think it's a common enough use case to include it maybe in the next PR when we update this.